.gitignore

```diff
@@ -165,9 +165,12 @@ package-lock.json
 
 # Allow for locally private items
 # private
-pri* 
+pri*
 # ignore
 ig*
 .github_access_token
 LICENSE.rtf
 autogpt_platform/backend/settings.py
+
+# Ignoring the supabase folder from pushing on github
+supabase/
```

Comment:
@aarushik93 is this a good idea? we can force push over when desired after all

Reply:
(merge if its having this in gitignore is okay)

Reply:
@Abhi1992002 can we not have this change please? If you could just not add your changes that should be enough